xml/System.Net.Security/NegotiateStream.xml

```diff
@@ -3073,10 +3073,24 @@ Authentication has not occurred.</exception>
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="0" FrameworkAlternate="net-5.0;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        <summary>Causes any buffered data to be asynchronously written to the underlying device.</summary>
+        <returns>A task that represents the asynchronous flush operation.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This method invokes <xref:System.IO.Stream.FlushAsync%2A> on the underlying stream.  
+  
+   
+  
+## Examples  
+ The following code example demonstrates flushing the stream.  
+  
+ [!code-csharp[NclNegoSyncClient#4](~/samples/snippets/csharp/VS_Snippets_Remoting/NclNegoasyncClient/CS/client.cs#4)]  
```

Comment:
The code sample isn't rendering in the [preview](https://review.docs.microsoft.com/en-us/dotnet/api/system.net.security.negotiatestream.flushasync?view=netcore-3.1&branch=pr-en-us-4335#examples). Also, I don't see a snippet4 in the file:

https://github.com/dotnet/dotnet-api-docs/blob/f8c76428f485dbc09e9c4fa8b30877fc44ac10d8/samples/snippets/csharp/VS_Snippets_Remoting/NclNegoasyncClient/CS/client.cs

Reply:
this section in the file is referred from multiple places. I will take a look if it renders correctly on other places too.

Reply:
there is quite inconsistency in the examples:
C++ and VB are same and uses BexinXXX, EndXXX methods
C# uses XXXAsync methods. 

[NclNegoasyncClient.cpp](https://github.com/dotnet/dotnet-api-docs/blob/fe23cadb370a6d719f318f561fe56875885e54e1/samples/snippets/cpp/VS_Snippets_Remoting/NclNegoasyncClient/CPP/NclNegoasyncClient.cpp)
[client.cs](https://github.com/dotnet/dotnet-api-docs/blob/fe23cadb370a6d719f318f561fe56875885e54e1/samples/snippets/csharp/VS_Snippets_Remoting/NclNegoasyncClient/CS/client.cs)
[client.vb](https://github.com/dotnet/dotnet-api-docs/blob/fe23cadb370a6d719f318f561fe56875885e54e1/samples/snippets/visualbasic/VS_Snippets_Remoting/NclNegoasyncClient/VB/client.vb)

It implies that in the BexinXXX method docs we show C# example however not for the specific api. On other side C++ and VB are accurate.

I'm not sure how to handle it as there is several options:
1) Create BexinXXX, EndXXX variant for C# async methods
     1. it demonstrate legacy approach
     2. we still need Task based async examples - code duplication
2) Let it like it is and remove C# example from  BexinXXX, EndXXX
     1. It sounds strange to me as C# is main .net language and we would offer examples in other languages only
3) Rework VB + C++ to not use BexinXXX, EndXXX but Task based async methods 
    1. It would unify the examples 
    2. it would require remove examples from  BexinXXX, EndXXX docs and move it to Task based async methods


Example of mismatched examples:
https://docs.microsoft.com/en-us/dotnet/api/system.net.security.negotiatestream.beginwrite?view=netcore-3.1

The C# example doesn't contain the related api.

@karelz, any thought?

Reply:
I removed the example completely. There is follow up ticket: #4390